Cargo.toml

```diff
@@ -20,12 +20,17 @@ license = "MPL-2.0"
 [features]
 default = []
 # Make Vulkan backend available on platforms where it is by default not, e.g. macOS
-vulkan = ["wgn/gfx-backend-vulkan"]
+vulkan = ["wgn/vulkan-portability"]
 
 [dependencies.wgn]
 package = "wgpu-native"
 version = "0.4"
-features = ["local"]
+path = "/hub/moz/wgpu/wgpu-native"
```

Comment:
Do we need to have the paths listed here?

Reply:
no, this entry had to change to a git link anyway, so I left it unchanged from my local override
fixing now